service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/DocmosisCommonElementDecorator.java

```diff
@@ -60,4 +60,10 @@ public DocmosisParameters decorate(DocmosisParameters currentParameters, CaseDat
             .courtseal(SEALED == status ? DocmosisImages.COURT_SEAL.getValue() : null)
             .build();
     }
+
+    private String getOrderTitle(DocmosisParameters currentParameters, Order orderType) {
```

Comment:
I know that I suggested this but I am beginning to think that it would be nicer to just set the title in each custom provider.

Reply:
Done, looks much nicer now setting it in the generators